app/templates/gitignore.tmpl

```diff
@@ -8,7 +11,7 @@ node_modules
 
 # Ignore config
 local-config.php
-<% if (conf.get('ignoreWPCore')) { %>
+<% if (conf.get('ignoreWPConfig')) { %>
 wp-config.php
```

Comment:
Do you think you could pull this change into another PR based on the existing master?  I would like to get this as a patch on the 1.4.X version as well.